src/Testing/TestCase.php

```diff
@@ -339,6 +339,21 @@ public function getFileHelper(): FileHelper
 		return self::getContainer()->getByType(FileHelper::class);
 	}
 
+	/**
+	 * Provides a DIRECTORY_SEPARATOR agnostic assertion helper, to compare file paths.
+	 *
+	 * @param string $expected
+	 * @param string $actual
+	 * @param string $message
+	 */
+	protected function assertSamePaths(string $expected, string $actual, string $message = ''): void
+	{
+		$expected = str_replace('/', DIRECTORY_SEPARATOR, $expected);
+		$actual = str_replace('/', DIRECTORY_SEPARATOR, $actual);
```

Comment:
Try $this->getFileHelper()->normalizePath() for the same result. Otherwise good idea!